packages/fx-core/src/plugins/solution/fx-solution/v2/utils.ts

```diff
@@ -328,6 +328,8 @@ export function fillInSolutionSettings(
     hostType = HostTypeOptionAzure.id;
   } else if (capabilities.includes(M365SearchAppOptionItem.id)) {
     capabilities = [MessageExtensionItem.id];
+    const scenarios = [M365SearchAppOptionItem.id];
+    answers[AzureSolutionQuestionNames.Scenarios] = scenarios;
```

Comment:
It looks like question model was used to pass information from solution to bot plugin, but it's not a real question, right?